iodata/test/test_charmm.py

```diff
@@ -0,0 +1,49 @@
+# IODATA is an input and output module for quantum chemistry.
+# Copyright (C) 2011-2019 The IODATA Development Team
+#
+# This file is part of IODATA.
+#
+# IODATA is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# IODATA is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+# --
+# pylint: disable=unsubscriptable-object
+"""Test iodata.formats.orcalog module."""
+
+from numpy.testing import assert_equal, assert_allclose
+
+from ..api import load_one
+
+try:
+    from importlib_resources import path
+except ImportError:
+    from importlib.resources import path
+
+
+def test_load_crambin():
+    # test gro file of one water
+    with path('iodata.test.data', 'crambin.crd') as fn_crd:
+        mol = load_one(str(fn_crd))
+    check_crambin(mol)
+
+
+def check_crambin(mol):
```

Comment:
There is no need for a separate check function, because it is not reused by other tests. Could you also add more assert lines, e.g. for a few elements of `atcoords` and the shapes of the values of the `atffparams` and `extra` attributes?

Reply:
No problem! Working on that.